docs/install.rst

```diff
@@ -111,6 +183,103 @@ When using conda, ffmpeg can be installed as follows::
     conda install -c conda-forge ffmpeg
 
 
+Conda environments
+------------------
+.. _PyCharm: https://www.jetbrains.com/pycharm/download/#section=windows
+.. _PyCharm documentation: https://www.jetbrains.com/help/pycharm/conda-support-creating-conda-virtual-environment.html
+.. _Jupyter: https://jupyter.org/
+.. _uninstall instructions: https://docs.anaconda.com/anaconda/install/uninstall/
+.. _VS Code: https://code.visualstudio.com/download
+.. _VS Code Environment Instructions: https://code.visualstudio.com/docs/python/environments#_conda-environments
+.. _numpy discussion: https://github.com/numpy/numpy/issues/15183#issuecomment-603575874
+
+If you have installed Pylake according to the Installation Instructions for Anaconda, then you should now have a separate environment for your Pylake work.
+You may be wondering why we needed to create a new environment for Pylake, and what an environment is.
+
+When using Python, you will quickly find that several packages that you can install depend on each other.
+For example, when installing Pylake, you could see that the installation of Pylake required numerous other packages to be installed as well.
+Most Python packages are continuously updated by their authors. Sometimes, these authors decide that certain existing functionalities need to be changed.
+This means that not all of them will be completely backwards compatible.
+Therefore, it can be challenging to find a set of packages and package versions that all work together.
+
+One pragmatic solution to this is to maintain separate Python environments for different projects.
+This means that you create independent "copies" of Python and its installed packages, so that the different projects you're working on don't interfere with each other.
```

Comment:
```suggestion
This means that you create independent "copies" of Python and its installed packages, so that the different projects you are working on do not interfere with each other.
```